amptorch/descriptor/MCSH/__init__.py

```diff
@@ -185,26 +186,35 @@ def calculate_fingerprints(self, atoms, element, log=None, calculate_derivatives
                     x_p, dx_p)
         
         if errno == 1:
-            print("ERROR: descriptor not IMPLEMENTED!!")
+            raise NotImplementedError("Descriptor not implemented!")
         # print(errno)
         fp = np.array(x)
-
         fp_prime = np.array(dx)
 
-        if "prime_threshold" in self.params_set:
-            threshold = self.params_set["prime_threshold"]
-            super_threshold_indices = np.abs(fp_prime) < threshold
-            print("threshhold: {} \tnum points set to zero:{} \t outof: {}".format(threshold, np.sum(super_threshold_indices), fp_prime.shape[0] * fp_prime.shape[1]))
-            fp_prime[super_threshold_indices] = 0.0
+        # print(fp)
+        # print(fp_prime)
+
+        # if "prime_threshold" in self.params_set:
+        #     threshold = self.params_set["prime_threshold"]
```

Comment:
Can you remove all the commented out logic in the MCSH files.

Reply:
I'll clean it up as much as I could,
though I would probably need to leave some there for debug purpose for now. once I'm convinced everything works as it should, I'll clean everything up

Reply:
Yup no worries if intentional, thought it was missed.